src/renderers/shared/utils/ReactErrorUtils.js

```diff
@@ -16,12 +17,17 @@ var caughtError = null;
 /**
  * Call a function while guarding against errors that happens within it.
  *
- * @param {?String} name of the guard to use for logging or debugging
+ * @param {String} name of the guard to use for logging or debugging
```

Comment:
In practice this is always called with a string. If I put it optional, then flow complains about the concatenation of `\`react-${name}\`` down below.

I chose to keep the types in the comments but we may want to get rid of them